sql/binding_contexts.cc

```diff
@@ -186,7 +189,44 @@ doGetFunction(const Utf8String & tableName,
                 std::make_shared<PathValueInfo>()};
     }
 
+    if (functionName == "columnPathElement") {
```

Comment:
I would add a test for those two in the pipeline executor. At least to be sure wheter it works or not. +1 after that.